vehicles.lua

```diff
@@ -1059,6 +1059,9 @@ function courseplay:getVehicleTurnRadius(vehicle)
 		if vehicle.maxTurningRadius then
 			turnRadius = vehicle.maxTurningRadius;
 			courseplay:debug(('%s -> TurnRadius: Using Giants maxTurningRadius: %.2fm'):format(nameNum(vehicle), vehicle.maxTurningRadius), courseplay.DBG_IMPLEMENTS);
+		elseif g_vehicleConfigurations:get(vehicle, 'turnRadius') then
```

Comment:
This is already done on line 1027

Reply:
Ok, didn't see that - I'll take it out

Reply:
I edited the section around 1027 a bit  - just a  cosmetic issue but it wrote the wrong radius to the debug-statement (which confused me several times)